services/http/api/lib/models/resource.ts

```diff
@@ -6,7 +6,7 @@ export type ResourceProperties = {
 }
 
 export type ResourceRelatedNodes = {
-    User: ModelRelatedNodesI<UserModel, UserInstance>
+    User: ModelRelatedNodesI<UserModel, UserInstance>;
```

Comment:
Again, I think we need some linting done here. I believe the standard is to avoid semicolon usage

Reply:
Linter fixed.